vllm/model_executor/models/internvl.py

```diff
@@ -613,6 +836,38 @@ def get_replacement_internvl(item_idx: int):
 
 
 class InternVLProcessingInfo(BaseInternVLProcessingInfo):
+    """InternVL ProcessingInfo extended for video processing"""
+
+    @property
+    def supports_video(self):
+        return getattr(self.get_hf_processor(), "supports_video", False)
```

Comment:
```suggestion
        return self.get_hf_processor().supports_video
```

Should be safe to access directly now. Also make this a cached property?